wapitiCore/attack/mod_methods.py

```diff
@@ -39,6 +39,48 @@ class ModuleMethods(Attack):
     do_post = True
     excluded_path = set()
 
+    async def test_compare_methods(self, request: Request, methods: set):
```

Comment:
You should think about renaming this. It is unclear because :
1 - It's not a test in the way it asserts nothing
2 - What will this function compare ?
3 - Is this only a comparison ? It returns a set of methods, and it is not reflected in the method name.